Description: install binaries to /usr/games and don't install changelog twice
Author: Michael Gilbert <michael.s.gilbert@gmail.com>
Index: minetest-0.4.3/src/CMakeLists.txt
===================================================================
--- minetest-0.4.3.orig/src/CMakeLists.txt	2012-09-05 00:31:06.000000000 +0200
+++ minetest-0.4.3/src/CMakeLists.txt	2012-09-25 18:11:08.613769178 +0200
@@ -370,7 +370,7 @@
 		set(ARCH i386)
 	endif()
 
-	set(CMAKE_CXX_FLAGS_RELEASE "-DNDEBUG ${RELEASE_WARNING_FLAGS} ${WARNING_FLAGS} ${OTHER_FLAGS} -O3 -ffast-math -Wall -fomit-frame-pointer -pipe -funroll-loops")
+	set(CMAKE_CXX_FLAGS_RELEASE "-DNDEBUG ${RELEASE_WARNING_FLAGS} ${WARNING_FLAGS} ${OTHER_FLAGS} -ffast-math -Wall -fomit-frame-pointer -pipe -funroll-loops")
 	set(CMAKE_CXX_FLAGS_DEBUG "-g -O1 -Wall ${WARNING_FLAGS} ${OTHER_FLAGS}")
 
 	if(USE_GPROF)
Index: minetest-0.4.3/cmake/Modules/FindIrrlicht.cmake
===================================================================
--- minetest-0.4.3.orig/cmake/Modules/FindIrrlicht.cmake	2012-09-05 00:31:06.000000000 +0200
+++ minetest-0.4.3/cmake/Modules/FindIrrlicht.cmake	2012-09-25 18:11:08.617769178 +0200
@@ -50,7 +50,7 @@
 		/usr/include/irrlicht
 	)
 
-	FIND_LIBRARY(IRRLICHT_LIBRARY NAMES libIrrlicht.a Irrlicht
+	FIND_LIBRARY(IRRLICHT_LIBRARY NAMES libIrrlicht.so
 		PATHS
 		/usr/local/lib
 		/usr/lib
